README.md

```diff
@@ -1,5 +1,13 @@
 # Alexa Skills SDK for .NET
 
+<table>
```

Comment:
We should use the GFM format for tables: https://guides.github.com/features/mastering-markdown/#syntax